crates/wasi-crypto/Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0"
-wasi-crypto = { path = "spec/implementations/hostcalls/rust", version = "0.1.5" }
+wasi-crypto = { git = "https://github.com/WebAssembly/wasi-crypto", rev="fa309ac", features = ["rcrypto"], default-features = false }
```

Comment:
wasmtime-wasi-crypto is published on crates.io, which I believe requires that all dependencies be published on crates.io. Can you publish this revision to a new crates.io release, and then depend on that?

Reply:
I'll ask.

Reply:
A new version has been pushed to `crates.io`.